app/services/migration/parity_check/client.rb

```diff
@@ -87,7 +91,11 @@ def url(app:)
     def formatted_path
       return path unless options[:id] && path.include?(":id")
 
-      path.sub(":id", eval(options[:id]).to_s) # rubocop:disable Security/Eval
+      id = eval(options[:id]) # rubocop:disable Security/Eval
```

Comment:
curious about this, why wouldn't we able to find an id 🤔 ideally all should be there?